opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -298,11 +297,15 @@ def get_meter(
         return DefaultMeter()
 
 
-MetricT = TypeVar("MetricT", Counter, ValueRecorder)
 InstrumentT = TypeVar(
-    "InstrumentT", Counter, UpDownCounter, Observer, ValueRecorder
+    "InstrumentT",
+    Counter,
+    UpDownCounter,
+    SumObserver,
+    UpDownSumObserver,
+    ValueObserver,
+    ValueRecorder,
```

Comment:
Is there some superclass for all instruments you can use here like `bound=Instrument`?

Reply:
Fixed to use `Metric` and `Observer`.